requirements-docker.txt

```diff
@@ -3,3 +3,8 @@ Click==7.0
 pyOpenSSL==19.1.0
 python-dotenv==0.10.3
 python-vagrant==0.5.15
+<<<<<<< HEAD
```

Comment:
Have you tested this code?

Reply:
Sorry, git issues again :( Have to get rid of the habit of pulling after rebasing....its fixed now. And yes, the code is teste (without the conflict lines of course)